src/crc-cli.ts

```diff
@@ -132,39 +130,3 @@ export async function getCrcVersion(): Promise<CrcVersion | undefined> {
 
   return undefined;
 }
-
-export async function daemonStart(): Promise<boolean> {
-  let command = getCrcCli();
-  let args = ['daemon', '--watchdog'];
-
-  const env = Object.assign({}, process.env); // clone original env object
-
-  // In production mode, applications don't have access to the 'user' path like brew
-  if (isMac() || isWindows()) {
-    env.PATH = getInstallationPath();
-  } else if (env.FLATPAK_ID) {
```

Comment:
:+1 removal is necessary as the state should only be controlled by the daemon, or the process that can start the daemon; in this case `crc setup`.